sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -902,6 +902,15 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
       .booleanConf
       .createWithDefault(true)
 
+  val ENABLE_GETJSONOBJECT_LEGACY = conf("spark.rapids.sql.getjsonobject.legacy.enabled")
```

Comment:
nit: Generally we use camelCase in the names of operations like this.

```
spark.rapids.sql.getJsonObject.legacy.enabled
```

But this is very minor, especially with it being an internal config.

Reply:
done.